docs/integrations/google_sheets.md

```diff
@@ -0,0 +1,67 @@
+# Google Sheets
+
+## Getting Started
+
+To use Google Sheets as a data source, you will need to install the `gspread` and `oauth2client` Python packages. You can install this package using `pip`:
+
+```bash
+pip install gspread oauth2client
+```
+
+## Authentication
+
+### Application Default Credentials (Recommended)
+
+The easiest way to authenticate with Google Sheets is to use [Application Default Credentials](https://cloud.google.com/docs/authentication/production). If you are running marimo on Google Cloud and your resource has a service account attached, then Application Default Credentials will automatically be used.
+If you are running marimo locally, you can authenticate with Application Default Credentials by running the following command:
+
+```bash
+gcloud auth application-default login
+```
+
+### Service Account Key File
+
+To authenticate with Google Sheets, you will need to create a service account and download the service account key file. You can create a service account and download the key file by following the instructions [here](https://cloud.google.com/iam/docs/creating-managing-service-account-keys).
```

Comment:
I think there's a slightly simpler way for accessing sheets data that doesn't require you to sign up for Google Cloud, via https://console.developers.google.com/. At least I can visit that from my personal email, maybe it's functionally the same. Read about that here: https://medium.com/@techno021/integrate-google-sheets-with-jupyter-notebook-e25a4c349828

Reply:
The link you attached is the same instructions (create a service account and key file)